modules/data.atmosphere/R/download.MACA.R

```diff
@@ -0,0 +1,145 @@
+##' Download MACA CMIP5 outputs for a single grid point using OPeNDAP and convert to CF 
+##' @name download.MACA
+##' @title download.MACA
+##' @export
+##' @param outfolder
+##' @param start_date
+##' @param end_date
+##' @param lat
+##' @param lon
+##' @param model , select which MACA model to run (options are CM3, ESM2M, ESM2G)
+##' @param scenario , select which scenario to run (options are rcp26, rcp45, rcp60, rcp85)
+##' @param ensemble_member , select which ensemble_member to initialize the run (options are r1i1p1, r3i1p1, r5i1p1)
```

Comment:
I don't know how best to handle this ... sorry ... but it would be great if we could avoid overloading terms like 'ensemble' within the workflow ... PEcAn has ecosystem model ensembles, and so do atmospheric models. I just looked up the conventions for identifying atomic datasets in 
the [CMIP5 reference syntax docs](https://github.com/PecanProject/pecan/files/473358/cmip5_data_reference_syntax_v0-25_clean.pdf) and its a quagmire ... important thing is to a) take advantage of their clearly defined directory and file naming structures while b) not confusing PEcAn users / developers. The only thing that occurs to me offhand would be to extend the name to include a reference to the data source like cmip5_ensemble, rcp_scenario, cmip5_model etc.